src/notifications/generate.ts

```diff
@@ -143,6 +147,16 @@ export const generateNotificationMap: Record<
       .avatarSource(ctx.source)
       .uniqueKey(ctx.bookmark.remindAt.toString())
       .objectPost(ctx.post, ctx.source, ctx.sharedPost),
+  streak_reset_restore: (builder, ctx: NotificationStreakContext) =>
+    builder
+      .icon(NotificationIcon.Streak)
+      .description('Click here if you wish to restore your streak')
+      .uniqueKey(format(ctx.streak.lastViewAt, 'dd-MM-yyyy'))
+      .targetUrl(notificationsLink)
+      .referenceStreak(ctx.streak)
+      .setTargetUrlParameter([
+        ['streak_restore', ctx.streak.currentStreak.toString()],
+      ]),
```

Comment:
`streak_restore` is something that triggers popup on the frontend on notifications page?

Reply:
Yes, it does!